src/core/operators/detect_text_in_image_tesseract.py

```diff
@@ -29,11 +30,11 @@ def state():
     pass
 
 
-# if __name__ == "__main__":
-#     initialize(param={})
-#     image_path = 'sample_data/hindi-text.png'
-#     text_data = run(image_path)
-#     print(text_data)
+if __name__ == "__main__":
```

Comment:
lets remove the if __name__ from the code

Reply:
Done!